packages/eas-cli/package.json

```diff
@@ -50,6 +50,7 @@
     "strip-ansi": "^6.0.0",
     "tar": "^6.0.5",
     "tempy": "^0.7.0",
+    "terminal-link": "^2.1.1",
```

Comment:
Please do not use terminal-link, It does not work correctly for some terminals  https://github.com/expo/eas-cli/issues/89